package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Please remove this file. Webpack wizard is supposed to work with latest libraries available, at any time.

Reply:
done